src/Animations/babylon.animation.ts

```diff
@@ -1,8 +1,21 @@
 ﻿module BABYLON {
+    /**
+     * Represents the range of an animation.

```

Comment:
No period at end of sentences :)

Reply:
All periods removed from ends of sentences in comments :)